docker-assets/httpd-environment.conf

```diff
@@ -1,5 +1,5 @@
 [Unit]
-After=initialize-httpd-auth.service
+After=initialize-httpd-auth.service wait-for-container-env.service
```

Comment:
I'd also add that to the After directive for the initialize-httpd-auth.service file since he also depends on container-environment. Thanks!!